port/zos390/omrsl.c

```diff
@@ -75,20 +76,14 @@ omrsl_open_shared_library(struct OMRPortLibrary *portLibrary, char *name, uintpt
 
 	Trc_PRT_sl_open_shared_library_Event1(openName);
 
-	/* dllload() can open a handle to the executable with its name itself (unlike NULL
-	 * on other platforms) provided the executable is itself built as a DLL (in XPLINK
-	 * mode), which anyway is required for enabling symbol resolution.
-	 */
-	handle = dllload(openName);
+	/* 0x002 = RTLD_NOW */
+	handle = dlopen(openName, 0x002);
```

Comment:
Is `RTLD_NOW` not defined by `<dlfcn.h>`?

Reply:
I thought so too, but I ran a build on zos, and it didn't recognize it...